services/gateway/apis/http-version3/swagger/definitions/dpos.json

```
@@ -70,6 +70,21 @@
             "consecutiveMissedBlocks": {
                 "type": "integer",
                 "example": 0
+            },
+            "forgedBlocks": {
+                "type": "integer",
+                "example": 1000,
+                "description": "Number of blocks forged by a delegate"
```

Let us make a practice of ending all the comments and descriptions with a full stop for consistency.
```suggestion
                "description": "Number of blocks forged by the delegate."
```